src/test/integration/configuration_example.json

```diff
@@ -0,0 +1,22 @@
+{
+    "hostname": "https://api.oregonstate.edu",
+    "version": "v1",
+    "api": "textbooks",
+    "use_basic_auth": false,
+    "ouath2": {
+        "token_api_url": "https://api.oregonstate.edu/oauth2/token",
+        "client_id": "client_id",
+        "client_secret": "client_secret"
+    },
+    "basic_auth": {
+        "username": "username",
+        "password": "password"
+    },
+    "valid_textbooks_parameters": {
```

Comment:
I'm a little hesitant to put this information in a configuration file since it seems like we'll have to update it every term or else these tests will fail. Is there a way to get this information dynamically from the textbooks API? I seem to remember verbacompare being able to produce this information, but I'm not sure if you were using any of those endpoints in the textbooks API. Maybe we need to have these integration tests get this information dynamically directly from verbacompare if the endpoints aren't present in the textbooks API?

Reply:
Makes sense. The textbooks api doesn't use verbacompare's /courses endpoint but I can have the tests query it